messaging/reader.go

```diff
@@ -94,7 +102,37 @@ func NewReader(brokers []string, groupID string, topic string) Reader {
 		MaxBytes:       10e6, // 10MB do we want it from config?
 	})
 
-	return &missyReader{brokers: brokers, groupID: groupID, topic: topic, brokerReader: &readBroker{kafkaReader}}
+	retries, err := strconv.Atoi(config.Get("kafka.retries.max.number"))
+	if retries <= 0 || err != nil {
+		log.Debug("Setting retries number to 3, as kafka.retries.max.number was not set or wrong")
+		retries = 3
+	}
+	var intervalTime time.Duration
+	interval, err := strconv.Atoi(config.Get("kafka.retries.interval.ms"))
+	if interval <= 0 || err != nil {
+		log.Debug("Setting retries interval to 5000 ms, as kafka.retries.interval.ms was not set or wrong")
+		intervalTime = 5000 * time.Millisecond
+	}
+
+	log.Infof("Configured num of maxRetries: %v with interval %v", retries, interval)
+
+	kafkaDlqTopicKey := fmt.Sprintf("kafka.%s.dlq_topic_name", topic)
```

Comment:
I think that we should add this info about dlq and about this configuration to a `README.md` file in a `Messaging` section.